### ISSUE TITLE
scenepic

### ISSUE THREAD

**Issue by Peter230655** [holzschu/Carnets#297]
I tried to install this library by writing
%pip install scenepic
but I got an error message.

Could this be so because it is not written in pure python?
If this is the case, I would request you to consider it to be included in your next update. It seems to be a very neat program for 3D animations.

Thanks!

**Comment by holzschu** [comment]
Yes, scenepic has a lot of source code written in C and C++, so it cannot be installed in Carnets with pip. I'll think about it.

**Comment by Peter230655** [comment]
Thanks!!

**Comment by 6zyman** [comment]
Can opencv be used on this application?

**Comment by holzschu** [comment]
OpenCV is, indeed, available on Carnets. You should try it and see if it works for your needs. The main issue is that it's better if the notebook and the image files are in the same directory.
